src/components/Chat/ChatMessage.tsx

```diff
@@ -44,21 +49,35 @@ class ChatMessage extends UIComponent<Extendable<IChatMessageProps>, any> {
     variables: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
   }
 
-  static handledProps = ['as', 'children', 'className', 'content', 'mine', 'styles', 'variables']
+  static handledProps = [
+    'as',
+    'avatar',
+    'children',
+    'className',
+    'content',
+    'mine',
+    'styles',
+    'variables',
+  ]
 
   static defaultProps = {
     as: 'li',
   }
 
-  renderComponent({ ElementType, classes, rest }) {
-    const { children, content } = this.props
+  renderComponent({ ElementType, classes, rest, styles }) {
+    const { avatar, children, content, mine } = this.props
 
     return (
       <ElementType {...rest} className={classes.root}>
-        {childrenExist(children) ? children : content}
+        {!mine && this.renderAvatar(avatar, styles)}
+        <div className={classes.chatContent}>{childrenExist(children) ? children : content}</div>
```

Comment:
there is an agreement that if `children` are provided in the following form (Children API)

```jsx
<ChatMessage>
   ... some child content
</ChatMessage> 
```

then we shouldn't wrap it in anything except `ElementType` node - these `children` should go 'as is'. Here we are breaking this rule. While I do see the point you are following with making this change, however, lets rather stick to the current approach taken, and reconsider it afterwards in case if needed. 

Please, let me know about your thoughts on that

Reply:
discussed with Roman and agreed to render `children` as is